lnrpc/walletrpc/walletkit.proto

```diff
@@ -524,13 +543,13 @@ message SignMessageWithAddrRequest {
     // base64.
     bytes msg = 1;
 
-    // The address which will be used to lookup the private key and sign the
-    // corresponding message
+    // The address which will be used to look up the private key and sign the
+    // corresponding message.
```

Comment:
Oh I see. So the changes need to happen in the exact commit where it's introduced. You may want to use `git commit --fixup` instead of `git commit --amend`.

Reply:
Ohh sorry, I thought I don't have to rearrange the fixups in the rebase process but it seems like you really have to, need to fix that now.

Reply:
should be fixed now